modules/microshift-low-latency-install-kernelrt-rhel-edge.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-install-kernelrt-rhel-edge_{context}"]
+= Installing the {op-system-rt-kernel} in a {op-system-ostree-first} image
+
+While the real-time kernel is not necessary for low latency workloads, using the real-time kernel can optimize low latency performance and is recommended. You can install it on a host using RPM packages, and include it in a {op-system-ostree-first} image deployment.
+
+.Prerequisites
+* You have a Red Hat subscription enabled on the host that includes {op-system-rt-kernel}.
+//do we need to have installed kernel-rt on the host before we can make the blueprint?
+* You are using the x86_64 architecture.
+//any other prereqs?
```

Comment:
osbuild needs to be configured to use kernel-rt repository. I need to recheck if https://osbuild.org/docs/on-premises/installation/managing-repositories/#using-repositories-that-require-subscription is still needed here

Reply:
I checked that and normal way of adding source to composer doesn't work. It needs what described in the link. 
https://redhat-internal.slack.com/archives/C0235DZB0DT/p1718982965408989?thread_ts=1718978986.534309&cid=C0235DZB0DT

Reply:
Looks like the RHEL doc is ready to be merged. So we should have a link in a day or two. I'll keep an eye on it.